onnxruntime/wasm/api.h

```diff
@@ -41,7 +41,7 @@ int EMSCRIPTEN_KEEPALIVE OrtInit(int num_threads, int logging_level);
  * @param enable_cpu_mem_arena enable or disable cpu memory arena
  * @param enable_mem_pattern enable or disable memory pattern
  * @param execution_mode sequential or parallel execution mode
- * @param enable_profiling enable or disable profiling. it's a no-op and for a future use.
+ * @param enable_profiling enable or disable profiling.
```

Comment:
is profiling per-session only?

Reply:
Native ORT can support multiple profiling in a session, but there is no exposed StartProfiling C API. Before that, it's per-session only now.